packages/design-tokens/config.js

```diff
@@ -0,0 +1,234 @@
+module.exports = {
+  source: ["src/**/*.tokens.json"],
+  platforms: {
+    css: {
+      transformGroup: "css",
+      prefix: "pluto",
+      buildPath: "dist/css/",
+      files: [
+        {
+          destination: "variables.css",
+          format: "css/variables",
+        },
+      ],
+    },
+    scss: {
+      transformGroup: "scss",
+      prefix: "pluto",
+      buildPath: "dist/scss/",
+      files: [
+        {
+          destination: "variables.scss",
+          format: "scss/variables",
+        },
+      ],
+    },
+    ts: {
+      transforms: ["attribute/cti", "name/cti/camel"],
+      transformGroup: "js",
+      buildPath: "dist/js/",
+      files: [
+        {
+          destination: "border-radius.js",
+          format: "javascript/es6",
+          filter: {
+            attributes: {
+              category: "border-radius",
+            },
+          },
+        },
+        {
+          destination: "border-radius.d.ts",
+          format: "typescript/es6-declarations",
+          filter: {
+            attributes: {
+              category: "border-radius",
+            },
+          },
+        },
+        {
+          destination: "border-style.js",
+          format: "javascript/es6",
+          filter: {
+            attributes: {
+              category: "border-style",
+            },
+          },
+        },
+        {
+          destination: "border-style.d.ts",
+          format: "typescript/es6-declarations",
+          filter: {
+            attributes: {
+              category: "border-style",
+            },
+          },
+        },
+        {
+          destination: "border-width.js",
+          format: "javascript/es6",
+          filter: {
+            attributes: {
+              category: "border-width",
+            },
+          },
+        },
+        {
+          destination: "border-width.d.ts",
+          format: "typescript/es6-declarations",
+          filter: {
+            attributes: {
+              category: "border-width",
+            },
+          },
+        },
+        {
+          destination: "colors.js",
+          format: "javascript/es6",
+          filter: (token) => {
+            return (
+              token.filePath === "src/tokens/color.tokens.json" &&
+              ["color", "bg"].includes(token.attributes.category)
```

Comment:
Rafa, do you know why they do not show here? https://632b1baf1257171b6a84f370-tppbulvbvh.chromatic.com/?path=/docs/docs-colors--docs